mindsdb/utilities/context_executor.py

```diff
@@ -57,3 +60,49 @@ def execute_in_threads(func, tasks, thread_count=3, queue_size_k=1.5):
 
         time.sleep(0.1)
     executor.shutdown(wait=False)
+
+
+def process_dataframe_in_partitions(df, callback, partition_size):
```

Comment:
@StpMax, not sure where is better to locate this function

Reply:
may be add new fine in `utilities/` folder?